{"name": "test_login_params[data_for_login1]", "status": "passed", "attachments": [{"name": "log", "source": "37a8cceb-38d5-4f1b-a730-1c64a6839b20-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data_for_login", "value": "('adelukesh1@gmail.com', '12345', 'Fail')"}], "start": 1743433483703, "stop": 1743433488079, "uuid": "ee40cd2e-c594-42ee-bd2f-fa2f4de47437", "historyId": "12f6c2828d495a13477c46fa3e65d5b0", "testCaseId": "47f970e602e4eefb1a3e5455dd555a36", "fullName": "testCases.test_login_params.TestLoginParams#test_login_params", "labels": [{"name": "severity", "value": "critical"}, {"name": "tag", "value": "regression"}, {"name": "tag", "value": "sanity"}, {"name": "tag", "value": "@pytest.mark.usefixtures('setup', 'log_on_failure')"}, {"name": "parentSuite", "value": "testCases"}, {"name": "suite", "value": "test_login_params"}, {"name": "subSuite", "value": "TestLoginParams"}, {"name": "host", "value": "lukesh"}, {"name": "thread", "value": "30484-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testCases.test_login_params"}]}